inc/packages/admin/namespace.php

```diff
@@ -36,6 +36,24 @@ function bootstrap() {
 	add_filter( 'plugin_install_action_links', __NAMESPACE__ . '\\maybe_hijack_plugin_install_button', 10, 2 );
 	add_filter( 'plugin_install_description', __NAMESPACE__ . '\\maybe_add_data_to_description', 10, 2 );
 	add_action( 'wp_ajax_check_plugin_dependencies', __NAMESPACE__ . '\\set_slug_to_hashed' );
+	add_filter( 'wp_list_table_class_name', __NAMESPACE__ . '\\maybe_override_list_table' );
+}
+
+/**
+ * Override the install list table with our own.
+ *
+ * @param string $class_name List table class name to use.
+ * @return string Overridden class name.
+ */
+function maybe_override_list_table( $class_name ) {
+	if ( $class_name !== 'WP_Plugin_Install_List_Table' ) {
+		return $class_name;
+	}
+
+	// Load list table class. (We must do this here, as WP_List_Table isn't loaded by default.)
```

Comment:
Is this confirmed?

Looking at [these lines](https://github.com/WordPress/wordpress-develop/blob/6.8.2/src/wp-admin/includes/list-table.php#L47-L50), it looks like the parent class should be loaded before the `wp_list_table_class_name` filters are applied.

It does no harm to have the `require_once` anyway. I was just curious if you got errors before doing so.